src/OpenLoco/src/Map/Track/TrackData.h

```diff
@@ -58,4 +58,8 @@ namespace OpenLoco::World::TrackData
     const stdx::span<const PreviewTrack> getRoadPiece(size_t trackId);
     const TrackCoordinates& getUnkTrack(uint16_t trackAndDirection);
     const TrackCoordinates& getUnkRoad(uint16_t trackAndDirection);
+
+    // TODO: Combine these two
```

Comment:
todo?

Reply:
Yeah something for a future PR. I suspect there are a few more meta data items that can be combined into this.